cmd/kubelet/app/server.go

```diff
@@ -371,6 +373,8 @@ func UnsecuredDependencies(s *options.KubeletServer) (*kubelet.Dependencies, err
 			return nil, err
 		}
 		mounter = mount.NewNsenterMounter(s.RootDirectory, ne)
+		// NSenter only valid on Linux
+		subpather = subpath.NewNSEnter()
```

Comment:
nit: I'd prefer if there was `subpath.NewNSEnter(ne)`, so you don't need to export Ne from the mounter.

Reply:
Make sense. I think it's a little weird to pass the `ne` object that will also be embedded in the `mounter` object that I am now going to pass as well, but it definitely keeps us from having to export it. will do.